src/components/slider/daikin-slider.ts

```diff
@@ -0,0 +1,306 @@
+    ${unsafeCSS(tailwindStyles)}
+
+    :host {
+      display: inline-flex;
+      width: 100%;
+    }
+  `;
+
+  /**
+   * The smallest value in the range of permitted values.
+   */
+  @property({ type: String, reflect: true })
+  min = "1";
+
+  /**
+   * The largest value in the range of permitted values.
+   */
+  @property({ type: String, reflect: true })
+  max = "100";
+
+  /**
+   * The step attribute is a number that specifies the granularity that the value must adhere to.
+   */
+  @property({ type: String, reflect: true })
+  step = "1";
+
+  /**
+   * The form name, submitted as a name/value pair when submitting the form.
+   */
+  @property({ type: String, reflect: true })
+  name = "";
+
+  /**
+   * The current value of the slider.
+   * Also used as a form value.
+   */
+  @property({ type: String, reflect: true })
+  value = this.min;
+
+  /**
+   * Specify the slider disabled state.
+   */
+  @property({ type: Boolean, reflect: true })
+  disabled = false;
+
+  /**
+   * The aria-label of the slider.
+   */
+  @property({ type: String, reflect: true, attribute: "slider-aria-label" })
+  sliderAriaLabel = "slider";
+
+  static readonly formAssociated = true;
+
+  private _sliderRef: Ref<HTMLElement> = createRef();
+
+  // define _internals to let the slider can be used in a form
+  private _internals = this.attachInternals();
+
+  private _updateFormValue() {
+    if (this.disabled) {
+      return;
+    }
+    this._internals.setFormValue(this.value);
+    this.dispatchEvent(
+      new Event("change", {
+        bubbles: true,
+        composed: true,
+      })
+    );
+    this.dispatchEvent(
+      new Event("input", {
+        bubbles: true,
+        composed: true,
+      })
+    );
+  }
+
+  private get _progress(): number {
+    const min = parseFloat(this.min);
+    const max = parseFloat(this.max);
+    const clampedValue = Math.max(min, Math.min(parseFloat(this.value), max));
+    return ((clampedValue - min) / (max - min)) * 100;
+  }
+
+  private _handleDrag = (event: MouseEvent | TouchEvent): void => {
+    const leftDistance = this._calcMousePositionRatio(event);
+    if (leftDistance == null) {
+      return;
+    }
+    const value = getValueFromRatio(this, leftDistance);
+    this.value = value;
+  };
+
+  // This function will triggered when click the slider bar area.
+  private _handleClick(event: MouseEvent) {
+    event.preventDefault();
+    if (this.disabled) {
+      return;
+    }
+    const ratio = this._calcMousePositionRatio(event);
+    if (ratio == null) {
+      return;
+    }
+    const value = getValueFromRatio(this, ratio);
+    this.value = value;
+  }
+
+  // This function is called when an arrow key is pressed while the thumb is focused.
+  private _handleKeyDown(event: KeyboardEvent) {
+    event.preventDefault();
+    if (this.disabled) {
+      return;
+    }
+    const value = getValueByKeydown(this, event.key);
+    if (value == null) {
+      return;
+    }
+    this.value = value;
```

Comment:
We should call `event.preventDefault();` only if the key stroke is handled.

```suggestion
    if (this.disabled) {
      return;
    }
    
    const value = getValueByKeydown(this, event.key);
    if (value == null) {
      return;
    }

    this.value = value;
    event.preventDefault();
```